walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java

```diff
@@ -26,25 +26,23 @@
 
   public static final TimeRange WHOLE_DAY = new TimeRange(0, 24 * 60);
 
-  /**
-   * A comparator for sorting ranges by their start time in ascending order.
-   */
-  public static final Comparator<TimeRange> ORDER_BY_START = new Comparator<TimeRange>() {
-    @Override
-    public int compare(TimeRange a, TimeRange b) {
-      return Long.compare(a.start, b.start);
-    }
-  };
-
-  /**
-   * A comparator for sorting ranges by their end time in ascending order.
-   */
-  public static final Comparator<TimeRange> ORDER_BY_END = new Comparator<TimeRange>() {
-    @Override
-    public int compare(TimeRange a, TimeRange b) {
-      return Long.compare(a.end(), b.end());
-    }
-  };
+  /** A comparator for sorting ranges by their start time in ascending order. */
+  public static final Comparator<TimeRange> ORDER_BY_START =
```

Comment:
nit:  I believe you can replace these anonymous inner classes with lambdas.

For example,
```java
public static final Comparator<TimeRange> ORDER_BY_START = (TimeRange a, TimeRange b) -> Long.compare(a.start, b.start);
```

Reply:
Fixed